src/AddIns/Uno.UI.MediaPlayer.Skia.Gtk/MediaPlayerExtension.events.cs

```diff
@@ -0,0 +1,183 @@
+﻿#nullable enable
+
+using System;
+using Uno.Media.Playback;
+using Windows.Media.Core;
+using Uno.Extensions;
+using System.IO;
+using Uno.Foundation.Logging;
+using System.Collections.Generic;
+using Uno;
+using Uno.Helpers;
+using System.Linq;
+using System.Threading.Tasks;
+using Windows.Media.Playback;
+using Windows.Storage;
+using Windows.Storage.Streams;
+using Windows.Foundation;
+using Windows.UI.Xaml.Controls;
+using System.Diagnostics.CodeAnalysis;
+using Windows.ApplicationModel.Background;
+using Uno.Foundation.Extensibility;
+using Windows.UI.Xaml.Controls.Maps;
+using System.Numerics;
+using Uno.Logging;
+using Windows.UI.Xaml;
+
+[assembly: ApiExtension(typeof(IMediaPlayerExtension), typeof(Uno.UI.Media.MediaPlayerExtension))]
+
+namespace Uno.UI.Media;
+
+public partial class MediaPlayerExtension : IMediaPlayerExtension
+{
+
+	private void OnStatusChanged(MediaPlaybackSession? sender, object args)
+	{
+		//if (this.Log().IsEnabled(LogLevel.Debug))
+		//{
+		//	this.Log().Debug($"OnStatusChanged: {args}");
+		//}
+
+		if ((MediaPlaybackState)args == MediaPlaybackState.Playing)
+		{
+			_player?.Play();
+		}
+		if ((MediaPlaybackState)args == MediaPlaybackState.None)
+		{
+			_player?.Stop();
+		}
+		if ((MediaPlaybackState)args == MediaPlaybackState.Paused)
+		{
+			_player?.Pause();
+		}
```

Comment:
Consider storing `(MediaPlaybackState)args` to a local variable, and using `else if`s